src/aind_slims_api/core.py

```diff
@@ -127,7 +127,7 @@ def fetch_models(
         start: Optional[int] = None,
         end: Optional[int] = None,
         **kwargs,
-    ) -> list[SlimsBaseModelTypeVar]:
+    ) -> List[SlimsBaseModelTypeVar]:
```

Comment:
Super trivial, and I could be wrong but isn't it now preferred to use `list` over `typing.List`?

Reply:
Ya, since the slims-api requires 3.10 and up, we can use the generic types. I'll also change the expected type for `fetch_attachments`